examples/websocket/example.rst

```diff
@@ -49,7 +49,7 @@ Step 2: Build and start the sandbox
 
 This starts three proxies listening on ``localhost`` ports ``10000-30000``.
 
-It also starts two upstream services, one ``ws`` and one ``wss``.
+It also starts four upstream services, ``ws``, ``ws-route``, ``wss`` and ``wss-passthrough``.
```

Comment:
the original is correct here - this is about _upstream_ services - ie backend websockets that envoy connects to.

Reply:
The bit that needs to change is above

> This starts three proxies listening on ``localhost`` ports ``10000-30000``.

Reply:
Hm. What need to change in this line? New test started on port 15000 (it's between 10000-30000).
May be need to start tests on 10000-20000-30000-40000 ports? (In my PR: 10000-15000-20000-30000)

Reply:
> This starts four proxies listening on localhost ports 10000, 15000, 2000, 30000.